src/test/java/de/hpi/bpt/argos/api/EventTypeEndpointTest.java

```diff
@@ -106,18 +106,18 @@ public void testCreateEventType() throws Exception {
     }
 
     @Test
-    public void testCreateEventType_EmptyJson_BadRequest() {
+    public void testCreateEventType_EmptyJson_Error() {
         RestRequest request = RestRequestFactoryImpl.getInstance()
                 .createPostRequest(ARGOS_REST_HOST, EventTypeEndpoint.getCreateEventTypeBaseUri());
 
         JsonObject emptyJson = new JsonObject();
         request.setContent(serializer.toJson(emptyJson));
 
-        assertEquals(HttpStatusCodes.BAD_REQUEST, request.getResponseCode());

```

Comment:
To me it's not a good idea to change this behavior. maybe just catch the parser exception and throw a halt?